src/html.pre.js

```diff
@@ -80,7 +81,7 @@ function pre(context) {
     .toArray();
   meta.description = `${desc.slice(0, 25).join(' ')}${desc.length > 25 ? ' ...' : ''}`;
   // url: use outer CDN URL if possible
-  meta.url = request.headers['x-cdn-url'] || `https://${request.headers.host}${request.url}`;
+  meta.url = `https://${getOriginalHost(request.headers)}${request.url}`;
```

Comment:
The `request.headers['x-cdn-url']` preference is now missing, is that deliberate?

Reply:
Yes 🙂 it should now use the exact same headers for the host name as the sitemap.xml.